.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.2
+ruby-2.3.0
```

Comment:
Couple of quick things here. Firstly the way rvm and rbenv parse this file means you can be more lax with it (so 2.3 would be preferable)

Also this should mirror the minimum ruby version of the gem (And what the codebase is developed against)

Reply:
Certainly, with you guidance on my problem bundling installing, i've cleared that out and am now running on ruby-2.2.

I have already pushed that up.